synapse/models/inet.py

```diff
@@ -3306,6 +3464,267 @@ def getModelDefs(self):
                             'ro': True,
                             'doc': 'The x509 certificate sent by the client.'})
                     )),
+                    ('inet:service:platform', {}, (
+
+                        ('url', ('inet:url', {}), {
+                            'ex': 'https://twitter.com',
+                            'doc': 'The primary URL of the platform.'}),
+
+                        ('name', ('str', {'onespace': True, 'lower': True}), {
+                            'ex': 'twitter',
+                            'doc': 'A friendly name for the platform.'}),
+
+                        ('provider', ('ou:org', {}), {
+                            'doc': 'The organization which operates the platform.'}),
+
+                        ('provider:name', ('ou:name', {}), {
+                            'doc': 'The name of the organization which operates the platform.'}),
+                    )),
+
+                    ('inet:service:instance', {}, (
```

Comment:
Do we want to add an :owner or :operator prop to capture the contact info of the persons who control the instance?